app/services/idv/profile_step.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class ProfileStep < Step
+    def complete?
+      idv_session.resolution.try(:success?) ? true : false
+    end
+
+    def complete
+      return if attempts_exceeded?
+      idv_session.params.merge!(params)
+      validate(params)
+      confirm if form_result[:success]
+      track_event
+      complete?
+    end
+
+    def attempts_exceeded?
+      attempter.exceeded?
+    end
+
+    private
+
+    def attempter
+      @_idv_attempter ||= Idv::Attempter.new(idv_form.user)
+    end
+
+    def confirm
+      idv_session.applicant = idv_session.applicant_from_params
+      idv_session.vendor = idv_agent.vendor
+      idv_session.resolution = idv_agent.start(idv_session.applicant)
+      attempter.increment
```

Comment:
It looks like the attempter is only being incremented if the basic info form fails vendor validation. Is that expected, or do we want to increment on each form that includes vendor validation, now that we have these distinct steps, each with its own vendor validation?